client/makelunch.js

```diff
@@ -82,12 +92,23 @@ Meteor.startup(function () {
 
   //registerHelpers
   UI.registerHelper('scoreSummary', Eaters.scoreSummary)
+
+  return SEO.config({
+    title: 'Makelunch',
+    meta: {
+      'viewport': 'initial-scale=1'
+    }
+  })
 })// end Meteor.startup
 
 UI.registerHelper('fromNow', function (date) {
   return moment(date + 'T12:00').fromNow()
 })
 
+UI.registerHelper('equal', function (a, b) {
```

Comment:
UI is an old alias for Template & Blaze, so the new hotness is to refer to the `registerHelper` function as `Template.registerHelper`
See: http://docs.meteor.com/#/full/template_registerhelper